DB/projects.json

```
@@ -150,6 +150,23 @@
         ]
       },
       {
+  "github_username": "https://github.com/AdityaShah7867",  
```

just add your github username not the link 

```suggestion
  "github_username": "AdityaShah7867",  
```